rules/cloud/policies/aws/s3/enable_bucket_logging.cf.go

```diff
@@ -1,44 +0,0 @@
-package s3
```

Comment:
We need to keep the examples (in Go) still available to us as they are used for doc generation. An example on how to use them is here https://github.com/aquasecurity/defsec/blob/master/rules/cloud/policies/aws/rds/disable_public_access.rego#L22-L25

Reply:
@simar7 The `good_example` field specifies the GO file, but the file also contains a bad example and links. How does it work?

Reply:
@nikpivkin they are loaded like such https://github.com/aquasecurity/defsec/blob/063dcd6ec8d061cae23ff208cf7a71c6eed6a8fc/pkg/rego/metadata.go#L266

Reply:
@simar7 I think I [found](https://github.com/aquasecurity/defsec/blob/master/cmd/avd_generator/main.go#L88) a code that parses GO files to generate documentation.

I have a question:
- Why are only good examples generated for documentation, but not bad ones?

Reply:
@simar7 By the way, the "RDS Publicly Accessible" rule is duplicated:
- rego https://github.com/aquasecurity/defsec/blob/master/rules/cloud/policies/aws/rds/disable_public_access.rego
- GO https://github.com/aquasecurity/defsec/blob/master/rules/cloud/policies/aws/rds/no_public_db_access.go

Reply:
> @simar7 By the way, the "RDS Publicly Accessible" rule is duplicated:
> 
> * rego https://github.com/aquasecurity/defsec/blob/master/rules/cloud/policies/aws/rds/disable_public_access.rego
> * GO https://github.com/aquasecurity/defsec/blob/master/rules/cloud/policies/aws/rds/no_public_db_access.go

New issue https://github.com/aquasecurity/trivy/issues/5065